tests/input/trailing_comma_on_array.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-$array =  [
```

Comment:
Don't think the `/input/` files should be changed here?

Reply:
@Ocramius The reason why I changed some inputs in to make them more unit. This one, for example, is testing if we have `trailing commas on arrays`, it has nothing to do with spaces before/after `=`.